Sources/BrowserServicesKit/PrivacyConfig/PrivacyConfigurationData.swift

```diff
@@ -112,15 +112,37 @@ public struct PrivacyConfigurationData {
             enum CodingKeys: String {
                 case state
                 case minSupportedVersion
+                case rollouts
             }
+            
+            public struct Rollout {
+                public let percent: Double
+                
+                enum CodingKeys: String {
+                    case percent
+                }
+                
+                public init(json: [String: Any]) {
+                    self.percent = json[CodingKeys.percent.rawValue] as? Double ?? 0
```

Comment:
Makes sense to fall back to `0` here :+1: I was wondering what would happen if there was an error parsing the rollout in the config (like if someone used `"5.0"` for the rollout value), and this way we won't make the feature visible unintentionally.